### ISSUE TITLE
Parsing error when angle brackets used

### ISSUE THREAD

**Issue by mattfysh** [styled-components/stylelint-processor-styled-components#181]
The following code causes the processor to silently fail:

```javascript
const styles = {
  a: css`
    content: '<';
  `,
  b: css`
    content: '>';
  `,
}
```

After some playing around, I found the angled brackets are matching and preventing the file from being properly passed.

1. is this a processor issue, or an upstream stylelint issue?
2. the silent failing could be masking other issues.. any way I can find out which files are silently failing, e.g. perhaps if I run in verbose/debug mode?

**Comment by chinesedfan** [comment]
@mattfysh Can you provide your lint config, the processor version and your sample test file? I tested following codes with a simple rule `"value-list-max-empty-lines": 1`, but correct errors are always reported, no matter I commented `const styles` or not.

```
import styled, { css } from 'styled-components';

const styles = {
  a: css`
    content: '<';
  `,
  b: css`
    content: '>';
  `,
}

const StyledTable = styled(StyledTableBase)`
    height: 100px;




    width: ${1


    };
`;
```

**Comment by mattfysh** [comment]
Upgrading stylelint from 9.2.0 to 9.2.1 has fixed this
